com.unity.render-pipelines.high-definition/Documentation~/Building-For-Consoles.md

```diff
@@ -1,10 +1,12 @@
 # Building your Project for consoles
 
-To build a Project for the **PlayStation 4**, **PlayStation 5** or **Xbox One**, you need to install an additional package for each platform you want to support. The packages for each platform are:
+To build a Project for the **PlayStation 4**, **PlayStation 5**, **Game Core Scarlett**, **Game Core Xbox One** or **Xbox One**, you need to install an additional package for each platform you want to support. The packages for each platform are:
```

Comment:
Is Scarlett a user facing name? Shouldn't this be Xbox One Series X/S ?

Reply:
@JordanL8

Reply:
Yes we should use the user-facing names here. So Xbox Series X/S instead of scarlett

Reply:
So currently the Unity Game Core editor extensions we provide to our users are called:

- Game Core - Scarlett
- Game Core - Xbox One

All of our documentation and C# defines also refer to the new Xbox as "Scarlett". So for consistency I'd like HDRP to keep using the Scarlett name in its documentation.

In the future we may move everything over to using "Xbox Series", but we'll do an en masse rename if this happens.